eclair-core/src/test/scala/fr/acinq/eclair/blockchain/electrum/ElectrumxService.scala

```diff
@@ -19,22 +19,22 @@ package fr.acinq.eclair.blockchain.electrum
 import com.spotify.docker.client.{DefaultDockerClient, DockerClient}
 import com.whisk.docker.impl.spotify.SpotifyDockerFactory
 import com.whisk.docker.scalatest.DockerTestKit
-import com.whisk.docker.{DockerContainer, DockerFactory}
+import com.whisk.docker.{DockerContainer, DockerFactory, LogLineReceiver}
 import org.scalatest.Suite
 
 trait ElectrumxService extends DockerTestKit {
   self: Suite =>
 
   val electrumxContainer = if (System.getProperty("os.name").startsWith("Linux")) {
     // "host" mode will let the container access the host network on linux
-    DockerContainer("lukechilds/electrumx")
+    DockerContainer("acinq/electrumx")
```

Comment:
why can't we use "lukechilds/electrumx"?

Reply:
Because it does not support version 1.4 of the Electrum protocol, which is more efficient and widely supported now. I'll try and see if we can work with 1.2